saltboot-formula/_states/saltboot.py

```diff
@@ -955,18 +955,28 @@ def _get_image_for_part(images, part):
     image_id = part.get('image')
     image_version = part.get('image_version')
     image_dict = None
+    sorted_versions = None
     if image_version is not None:
         try:
             if image_version in images[image_id]:
+                    # version includes release: A.B.C-R
                     image_dict = images[image_id][image_version]
```

Comment:
Nitpick: can you make indentation consistent? This is one level deeper then should be.